django_app/redbox_app/redbox_core/management/commands/create_admin_user.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand
+
+USER = get_user_model()
+logger = logging.getLogger(__name__)
+logger.setLevel("DEBUG")
+
+
+class Command(BaseCommand):
+    help = """This should be run once per environment to set the initial superuser.
+    Thereafter the superuser should be able to login and assign new staff users via the admin.
+    """
+
+    def handle(self, *args, **kwargs):
```

Comment:
Should we make this a migration and set the setting to non-nullable? e.g. make it a system setup command, so people can't find themselves locked out of their system